core/admin/admin.php

```diff
@@ -695,6 +695,7 @@ public function admin_action_new_post() {
 		}
 
 		$post_data = Utils::get_super_global_value( $_GET, 'post_data' ) ?? [];
+		$post_data['post_title'] = urldecode( Utils::get_super_global_value( $_GET, 'post_title' ) ) ?? null;
```

Comment:
@mykytamurzin As discussed, see if you can remove this change and make it part of `$_GET['post_data']` 🙏